maxim/apis/maxim_apis.py

```
@@ -1,6 +1,8 @@
 import contextlib
 import json
+import mimetypes
 import time
+from collections import defaultdict
```

Ruff (0.12.2)

`collections.defaultdict` imported but unused

Remove unused import: `collections.defaultdict`

(F401)